idxmgmt/ilm/policy.go

```diff
@@ -17,69 +17,75 @@
 
 package ilm
 
-import "github.com/elastic/beats/libbeat/common"
+type m map[string]interface{}
```

Comment:
We should start considering something like https://github.com/elastic/hey-apm/blob/master/types/types.go, instead of repeating this all over the place

Reply:
I don't think that this should be a public type. You argue for not having abbreviations, having a public type `M` or similar is worse IMO.

Reply:
I'm arguing for readability, and I think we saw enough of those to know what it means without going to the type definition. If it is public it can be called `Map` (a better name than `m`), but if it is private it can't be called `map`.

So why is worse to declare it once and reuse it everywhere than declaring it each time of use?